ale_linters/go/golangserver.vim

```diff
@@ -0,0 +1,19 @@
+" Author: Horacio Sanson <https://github.com/hsanson>
+" Description: Support for go-langserver https://github.com/sourcegraph/go-langserver
+
+call ale#Set('go_langserver_executable', 'go-langserver')
+
+function! ale_linters#go#golangserver#GetCommand(buffer) abort
+    let l:executable = ale#Var(a:buffer, 'go_langserver_executable')
+    let l:completion_enabled = ale#Var(a:buffer, 'completion_enabled')
+    let l:exec_args = l:completion_enabled == 1 ? ' -gocodecompletion' : ''
```

Comment:
Could you add a general `go_langserver_flags` variable? There are several options that may be useful to set by the user.